src/Model/User.php

```diff
@@ -96,6 +96,36 @@ class User
 
 	private static $owner;
 
+	private static function welcome_text(): string
+	{
+		return Strings::deindent(DI::l10n()->t('
+		The login details are as follows:
+
+		Site Location:	%1$s
+		Login Name:		%2$s
```

Comment:
Switch to nowdoc instead, so the translation string won't contain the `\t` tabs later and we don't need to call `Strings::deindent()`.

```suggestion
		$body = DI::l10n()->t(<<<'TEXT'
			The login details are as follows:

			Site Location:	%1$s
			Login Name:		%2$s
			[…]
			TEXT;
```

Reply:
I was also stating to wonder if the tabs would be gone if `String::deindent` was done before `l10n()->t` instead of after...?

I didn't immediately try it out because then new translations will be needed, and right now I'm reusing one of the existing ones. Or at least I thought I was, but maybe not.

Will change it to `nowdoc`.

Thanks for the review and feedback!

Reply:
Should I do these changes myself or apply your suggestions? Not sure what you usually do.

Reply:
@mfxa Please do these changes yourself. My suggestions are incomplete.